CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 0.6.1 - [2019-12-05]
+### Fixes
+- Fix Cloudformation conditions which could make to not evaluate them as bool. 
```

Comment:
Hmm could this be made clearer - `which could make to not` is the confusing bit for me. Something like:

`Fix CloudFormation conditions which were logically boolean to now successfully be evaluated as boolean.`

I might have got the meaning wrong :p